scalyr_agent/builtin_monitors/kubernetes_events_monitor.py

```diff
@@ -78,11 +74,14 @@
                      'absolute path',
                      convert_to=str, default='kubernetes_events.log')
 
-define_config_option( __monitor__, 'event_object_filter',
-                     'Optional (defaults to [ \'CronJob\', \'DaemonSet\', \'Deployment\', \'Job\', \'Node\', \'Pod\', \'ReplicaSet\', \'ReplicationController\', \'StatefulSet\' ] ). A list of event object types to filter on. '
-                     'If set, only events whose `involvedObject` `kind` is on this list will be included.',
-                     default=[ 'CronJob', 'DaemonSet', 'Deployment', 'Job', 'Node', 'Pod', 'ReplicaSet', 'ReplicationController', 'StatefulSet' ]
-                     )
+EVENT_OBJECT_FILTER_DEFAULTS = [
+    'CronJob', 'DaemonSet', 'Deployment', 'Job', 'Node', 'Pod', 'ReplicaSet', 'ReplicationController', 'StatefulSet'
```

Comment:
Refactor default `event_object_filter` list to prevent divergence from description